setup.py

```diff
@@ -56,13 +96,16 @@ def extract_version():
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Topic :: Scientific/Engineering',
         'Topic :: Scientific/Engineering :: GIS',
     ],
     extras_require={'test': ['nose']},
-    test_suite='{}.tests'.format(NAME),
+    setup_requires=requirements,
```

Comment:
@tjtg So given the awesomeness of PEP 517 and PEP 518, then surely the setup_requires argument is now redundant here?

Reply:
I think this is partly but not completely redundant. `pyproject.toml` provides information about what packages are needed to be able to run setup.py, but it's possible for other packages to be needed to actually run the the setup process.

I think it's better to keep this, even if it's somewhat duplicated or redundant.

Reply:
Agreed. Seems prudent to keep it, thanks.